ambari_plugin/README.md

```diff
@@ -16,8 +17,12 @@
 3. Install with rpm package
 
    - Manual installation (recommended）：
-      - Copy the prepared RPM packages to each node of the cluster
-      -  Execute with DS installation user: ```rpm -ivh apache-dolphinscheduler-incubating-xxx.noarch.rpm```
+      - Copy the prepared RPM packages to each node of the cluster.
+      - Execute with DS installation user: ```rpm -ivh apache-dolphinscheduler-incubating-xxx.noarch.rpm```
+      - Mysql-connector-java packaged using the default POM file will not be included.
+      - The RPM package was packaged in the project with the installation path of /opt/soft. 
+        If you use mysql as the database, you need add it manually.
```

Comment:
Is it better to combine `Mysql-connector-java packaged using the default POM file will not be included.` and `If you use mysql as the database, you need add it manually.`?

Reply:
Yeah. 
Mysql's non-Apache protocol does not provide the JAR package containing the mysql-Connector. When making the RPM package with the plug-in in the project, the POM file can be modified to include the lib mysql-Connecto in the RPM package